test/test-network/systemd-networkd-tests.py

```diff
@@ -91,6 +91,23 @@ def f(func):
 
     return f
 
+def expectedFailureIfLinkFileFieldIsNotSet():
+    def f(func):
+        support = False
+        rc = call('ip link add name dummy99 type dummy')
+        if rc == 0:
+            ret = run('udevadm info -w10s /sys/class/net/dummy99', stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
this is racy, the event from the kernel might not have reached udev yet by the time you wait on it here... 

i wonder if we maybe should introduce "udevadm wait" or so, where you can specify a device and it waits (with a timeout) until the device shows up in the udev db if its's not there. This is a lot more correct then the "settle" stuff people usually try...

Reply:
I've already add `-w` option for that purpose.